packages/client/tests/functional/raw-queries/typed-results/tests.ts

```diff
@@ -6,17 +7,27 @@ declare let prisma: PrismaClient
 declare let Prisma: typeof PrismaNamespace
 
 testMatrix.setupTestSuite(
-  ({ clientRuntime, driverAdapter }) => {
+  ({ clientRuntime, driverAdapter, provider }) => {
+    beforeEach(async () => {
+      await prisma.testModel.deleteMany()
+    })
+
+    function getAllEntries() {
+      if (provider === Providers.MYSQL) {
+        return prisma.$queryRaw`SELECT * FROM \`TestModel\`;`
+      } else {
+        return prisma.$queryRaw`SELECT * FROM "TestModel";`
+      }
+    }
+
     test('simple expression', async () => {
       const result = (await prisma.$queryRaw`SELECT 1 + 1`) as Array<Record<string, unknown>>
       expect(Number(Object.values(result[0])[0])).toEqual(2)
     })
 
-    // TODO: The buffer binary data does not match the expected one
-    skipTestIf(clientRuntime === 'wasm')('query model with multiple types', async () => {
+    test('query model with multiple types', async () => {
```

Comment:
Instead of skipping the test for "wasm", I made it ignore the result for `bytes` for "wasm", see `expect.anything()` below